MyConbook/src/main/java/net/myconbook/android/ui/elements/RestaurantListItem.java

```diff
@@ -105,7 +105,7 @@ public void populateViewHolder(Holder holder) {
         // Redo name
         String name = getName();
         if (getOpenStatus() == RestaurantOpenStatus.Closed) {
-            name += " [Closed]";
+            name += getString(R.string.brackets_closed);
         }
 
         holder.name.setText(name);

```

Comment:
Below here is `dollars += "D"` single character for `hasDelivery()`. Did you want this as a string resource as well?